eng/scripts/Get-TypeSpec-Folders.ps1

```diff
@@ -9,7 +9,7 @@ param (
 )
 
 $changedFiles = @()
-$allChangedFiles = (Get-ChildItem -path ./specification tspconfig.yaml -Recurse).Directory.FullName | ForEach-Object {[IO.Path]::GetRelativePath($($pwd.path), $_)}
+$allChangedFiles = (Get-ChildItem -path ./specification tspconfig.* -Recurse).Directory.FullName | ForEach-Object {[IO.Path]::GetRelativePath($($pwd.path), $_)}
```

Comment:
Keep this part of the change, since it's necessary to treat folders containing `tspconfig.yml` as "TypeSpec Folders".